ocaml/testsuite/tests/typing-implicit-source-positions/invalid_usages.ml

```diff
@@ -0,0 +1,49 @@
+(* TEST
+   * expect
+*)
+
+type t = [%src_pos]
+[%%expect {|
+Line 1, characters 11-18:
+1 | type t = [%src_pos]
+               ^^^^^^^
+Error: Uninterpreted extension 'src_pos'.
+|}]
+
```

Comment:
Let's not get distracted now, but leave a CR here that we should improve that error message to tell the user that `[%src_pos]` can be used only in arguments.